tests/Users.IntegrationTests/UserStatusTest.cs

```diff
@@ -0,0 +1,63 @@
+using ActualChat.Host;
+using ActualChat.Testing.Host;
+using Microsoft.Extensions.Configuration;
+
+namespace ActualChat.Users.IntegrationTests;
+
+public class UserStatusTest : AppHostTestBase
+{
+    private static readonly UserStatus NewUserDefaultStatus = UserStatus.Active;
+
+    private WebClientTester _tester = null!;
```

Comment:
We typically don't separate props & fields w/ empty lines:
- They seem more readable as a block
- Besides that, the more you can see - the better (assuming you don't sacrifice readability).

Reply:
done